src/library/scala/collection/convert/Wrappers.scala

```diff
@@ -411,24 +411,21 @@ private[collection] trait Wrappers {
 
     override def size = underlying.size
 
-    def get(k: String) = {
-      val v = underlying get k
-      if (v != null) Some(v.asInstanceOf[String]) else None
+    def get(k: String): Option[String] = {
+      Option((underlying get k).asInstanceOf[String])
```

Comment:
Are you sure that the `Option(x)` pattern is as efficient as the `if (x != null) Some(x) else None` pattern?  On something as basic as `get`, we _do_ care.